tests/pessimistictest/pessimistic_test.go

```diff
@@ -158,7 +158,7 @@ func createMockStoreAndSetup(t *testing.T, opts ...mockstore.MockTiKVStoreOption
 	return store, func() {
 		dom.Close()
 		require.NoError(t, store.Close())
-		transaction.PrewriteMaxBackoff = 20000
+		transaction.PrewriteMaxBackoff.Store(20000)
```

Comment:
It should be safe already after `dom.Close`?
No transaction should be using `transaction.PrewriteMaxBackoff` unless there is a leaked goroutine.

Reply:
Yes, I agree. but this is the mistake of design. Client-go classmate says ```PrewriteMaxBackoff``` is a parameter to use in the test so that it is public to use. it should consider concurrent scenarios to avoid the data race. Become atomic, which can prevent this kind of problem from happening.